superset/config.py

```diff
@@ -260,10 +260,10 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 # ---------------------------------------------------
 # Roles config
 # ---------------------------------------------------
-# Grant public role the same set of permissions as for the GAMMA role.
+# Grant public role the same set of permissions as for a selected builtin role.
 # This is useful if one wants to enable anonymous users to view
 # dashboards. Explicit grant on specific datasets is still required.
-PUBLIC_ROLE_LIKE_GAMMA = False
+PUBLIC_ROLE_LIKE = "Gamma"
```

Comment:
This probably shouldn't default to "Gamma" - the old default behavior was to have a public role with no permissions.

Reply:
+1, maybe `None` and handling that as noop